docs/examples/example.json

```
@@ -986,7 +986,11 @@
       "environment": {
         "goals": [
           "climateChangeMitigation"
-        ]
+        ],
+        "abatementCost": {
+          "amount": 50000,
```

Minor point, but the example abatement costs that I found when researching this field were much smaller than 50,000 USD.

```suggestion
          "amount": 12.29,
```